### PR TITLE
[functions] add `getEnv` method

### PR DESCRIPTION
**Pull request by Kikobeats**
The new method `getEnv()` makes easy to interact with [Vercel System Environment Variables](https://vercel.com/docs/projects/environment-variables/system-environment-variables#system-environment-variables-overview)

CleanShot 2024-07-08 at 19 37 42@2x

### CODE REVIEW

File: packages/functions/src/system-environment-variables.ts [hunk 1]
```diff
@@ -0,0 +1,188 @@
+export const systemEnvironmentVariables = (
```
Comment (Schniz) [inline packages/functions/src/system-environment-variables.ts]: maybe `getEnv`? <picture data-single-emoji=":eye-twitch:" title=":eye-twitch:">:eye-twitch:</picture>

**Review by Schniz — DISMISSED** [review]
<picture data-single-emoji=":lgtm:" title=":lgtm:">:lgtm:</picture>

i just wonder if the name of the function is good or we should make it shorter

no strong feelings for discrete functions vs returning a big "env var" record

File: packages/functions/src/system-environment-variables.ts (hunk 1 above)
Comment (Kikobeats) [reply packages/functions/src/system-environment-variables.ts]: `systemEnv` ?

**Review by Schniz — DISMISSED** [review]
<picture data-single-emoji=":lfgtm-intense:" title=":lfgtm-intense:">:lfgtm-intense:</picture>

**Review by Schniz — APPROVED** [review]

**Pull request merged**